### PR TITLE
Add docstrings to python3 and python2 buildpacks

### PR DESCRIPTION
**Pull request by willingc**
Addresses #211.

### CODE REVIEW

File: repo2docker/buildpacks/python/__init__.py [hunk 1]
```diff
@@ -1,39 +1,80 @@
...
+"""Generates Dockerfiles based on an input matrix based on Python."""
 import os
+
 from ..base import BaseImage
 
 
 class PythonBuildPack(BaseImage):
+    """Setup Python 3 for use with a repository."""
     def get_packages(self):
+        """Return list of Python3 packages to be installed."""
```
Comment (betatim) [inline repo2docker/buildpacks/python/__init__.py]: "Return list of system-level Python3 dependencies" (I don't like that sentence that much). Something to clarify that these are the apt-get packages we install, not python packages (like say numpy).

**Comment by betatim** [comment]
Looks good! One comment about system packages vs python packages.

File: repo2docker/buildpacks/python/__init__.py (hunk 1 above)
Comment (willingc) [reply repo2docker/buildpacks/python/__init__.py]: Good distinction @betatim. Will update.

**Comment by willingc** [comment]
Thanks folks.

**Pull request merged**